python/ray/train/v2/_internal/state/state_manager.py

```diff
@@ -59,10 +62,18 @@ def create_train_run(
     def update_train_run_scheduling(
         self,
         run_id: str,
+        resize_decision: Optional[ResizeDecision] = None,
     ) -> None:
+        if resize_decision is not None:
+            status_detail = _get_scheduling_status_detail(
+                resize_decision.num_workers, resize_decision.resources_per_worker
+            )
+        else:
+            status_detail = None
```

Comment:
should we also report the status detail when the decision is not a resize? 
e.g., something like `running with N workers`

Reply:
This is just for the `SCHEDULING` state which means that it is not running yet.